src/backend/src/services/projects.services.ts

```diff
@@ -78,15 +78,17 @@ export default class ProjectsService {
     carNumber: number,
     name: string,
     summary: string,
-    teamId: string | undefined
+    teamIds: string[]
   ): Promise<WbsNumber> {
     if (isGuest(user.role)) throw new AccessDeniedGuestException('create projects');
 
     await validateChangeRequestAccepted(crId);
 
-    if (teamId) {
-      const team = await prisma.team.findUnique({ where: { teamId } });
-      if (!team) throw new NotFoundException('Team', teamId);
+    if (teamIds.length > 0) {
```

Comment:
you're not actually adding the teamIds to the created project rn

Reply:
yeah this is just validating that all the teamIds map to a team in the db

Reply:
oh i see what your saying nvm i fixed it